app/views/rooms/location.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= provide(:title, "Location") %>
+<div class="row mx-4">
+    <div class="col-2">
+        <%= render 'rooms/menu' %>
+    </div>
+    <div class="col-10">       
+        <div class="card container d-flex justify-content-center mt-4">
+            <div class="card-header">
+                <h3>Location<h3>
+            </div>
+            <%= form_for(@room) do |f| %>
+            <div class="card-body">
+                <div class="form-group mx-4">
+                    <%= f.label :Address %>
+                    <%= f.text_field :is_active, class: "form-control" %>
```

Comment:
there is no `address` column in your migration, and this is displaying true instead of the address i typed..
this should be using the `address` field in the migration

so please update your room migration to fix this issue